lib/Reader/abi.cpp

```diff
@@ -29,9 +29,9 @@
 using namespace llvm;
 
 // Static class with helpers for the Microsoft x86-64 ABI.
-class X86_64_Win64 {
+class XArchWin64 {
```

Comment:
Is this really flagged? LLVM itself uses this convention when referring to x86-64. Is there any way to exclude this particular term?

In any case, please do not change `X86_64` to `XArch`: instead, please use `AMD64` or similar.